packages/kitchen-sink/src/examples/password-strength-analyser.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect } from 'react';
+
+function PasswordStrengthIndicator() {
```

Comment:
Hey there, I don't see any usage of Million.js anywhere here 😁. Could you maybe review this?